include/irr/switch_constexpr.h

```diff
@@ -2,8 +2,8 @@
 // This file is part of the "Nabla Engine".
 // For conditions of distribution and use, see copyright notice in nabla.h
 
```

Comment:
@Crisspl & @Przemog1 do we even use this anywhere?

Reply:
I don't recall

Reply:
we can delete the file then